src/index.js

```diff
@@ -4,7 +4,7 @@ import { findDOMNode } from 'react-dom';
 import throttle from 'lodash.throttle';
 import raf from 'raf';
 import getDisplayName from 'react-display-name';
-import { Consumer as DragDropContextConsumer } from 'react-dnd/lib/DragDropContext';
+import { Consumer as DragDropContextConsumer } from 'react-dnd/lib/cjs/DragDropContext';
```

Comment:
We can also import by the direct react-dnd path
import { Consumer as DragDropContextConsumer } from 'react-dnd'; 

In this way both react-dnd versions (7.0.2 the older and 7.2.0 the latest) will work.

Reply:
Unfortunately Consumer is not exported from react-dnd

Reply:
@dolezel : Its exported from react-dnd, I tried to build it on my machine its working perfect. 
<img width="1019" alt="screenshot 2019-03-08 at 3 19 56 pm" src="https://user-images.githubusercontent.com/2504495/54022235-07abd380-41b8-11e9-9ed7-caccc9a9694e.png">

Reply:
You are looking on compiled content of DragDropContext.tsx
if you check what is exported from react-dnd (`console.log(require('react-dnd'))`), it is this: 
```
DragDropContext: ƒ DragDropContext(backendFactory, backendContext, debugMode)
DragDropContextProvider: ƒ (_a)
DragLayer: ƒ DragLayer(collect, options)
DragSource: ƒ DragSource(type, spec, collect, options)
DropTarget: ƒ DropTarget(type, spec, collect, options)
__esModule: true
__proto__: Object
```